src/webui/components/browser_settings_tab.py

```
@@ -1,11 +1,13 @@
 import os
+import platform
 
 import gradio as gr
 import logging
 from gradio.components import Component
```

Ruff (0.11.9)

`gradio.components.Component` imported but unused

Remove unused import: `gradio.components.Component`

(F401)